MetaMask/metamask-extension
# Added Tag Url Component

* Fixes #15104 

### After

#### Without CTA
<img width="624" alt="Screenshot 2022-10-13 at 10 35 09 PM" src="https://user-images.githubusercontent.com/39872794/195660492-1c50400f-4fdc-4234-8f53-6c65c37bfe03.png">

#### With CTA
<img width="1287" alt="Screenshot 2022-10-13 at 10 36 16 PM" src="https://user-images.githubusercontent.com/39872794/195660962-49d03c1e-9b07-48d6-909b-0a3923d18738.png">

#### Overflow Text
<img width="599" alt="Screenshot 2022-10-13 at 10 37 49 PM" src="https://user-images.githubusercontent.com/39872794/195661249-4a31da4a-02bd-434a-beb2-802a7b92bdae.png">

## Manual Testing Steps

Storybook
- Go to the latest build of the storybook in this PR
- Search for TagUrl in component-library
- Check the stories, controls, and documentation

## Pre-Merge Checklist


### + If there are functional changes:
